testData/ontologies/mappings/ISO24751-flat.json5

```diff
@@ -54,6 +54,7 @@
     "http://registry\\.gpii\\.net/common/simplifiedUi/enabled": "-provisional-cognitiveSupport.-provisional-simplifiedUi/enabled",
     "http://registry\\.gpii\\.net/common/syllabification/enabled": "-provisional-cognitiveSupport.-provisional-syllabification/enabled",
     "http://registry\\.gpii\\.net/common/volume": "-provisional-general.-provisional-volume",
+    "http://registry\\.gpii\\.net/common/brightness": "-provisional-general.-provisional-brightness",
```

Comment:
If we are being consistent this should be "display.screenEnhancement.-provisional-screenBrightness" although I think there is now nothing left in the system that reads this file

Reply:
Yes, changing it. I don't think there is, but just for the sake of consistency.